docs: pre-existing TS error in zfb.config.ts — copyPublicWithBase not in ZfbConfig type at next.35

## Summary

`docs/zfb.config.ts` uses `copyPublicWithBase: false` (line 140), but `@takazudo/zfb@0.1.0-next.35` (currently installed) does not include this property in its `ZfbConfig` type.

This causes `pnpm --filter docs check` to fail with:

```
zfb.config.ts(140,3): error TS2353: Object literal may only specify known properties, and 'copyPublicWithBase' does not exist in type 'ZfbConfig'.
```

## Context

- `copyPublicWithBase` was added to `zfb.config.ts` in the base branch start commit (77a3b93 — `= start rebuilds-docs020 dev =`) as part of the rebuilds-docs020 epic setup.
- The property was merged into `main` via `deferred-decisions/copy-public` but the docs still use `@takazudo/zfb@0.1.0-next.35` which predates the type update.
- The fix is either a zfb version bump that includes the `copyPublicWithBase` type, or a guard/cast in `zfb.config.ts` until the bump lands.

## Discovered by

Agent during docs020-bump (#938) implementation in the rebuilds-docs020 epic (#937). The error is pre-existing (exists on the base branch before any changes from #938).

Parent epic: #937

--- a/docs/zfb.config.ts
+++ b/docs/zfb.config.ts
@@ -137,6 +137,9 @@ export default defineConfig({
   // public/ to dist/ (not to dist/pj/zudo-front-builder/) is the
   // correct placement. copyPublicWithBase:false replaces the old
   // copy-public-plugin.mjs that did the same thing manually.
+  // @ts-expect-error -- copyPublicWithBase is honored by the local engine on main but
+  // missing from the pinned @takazudo/zfb 0.1.0-next.35 ZfbConfig type (#946); this
+  // directive errors (forcing its removal) once the bump to a typed version lands.
   copyPublicWithBase: false,
   collections,
   stripMdExt: true,

--- a/docs/zfb.config.ts
+++ b/docs/zfb.config.ts
@@ -140,6 +140,7 @@ export default defineConfig({
   // @ts-expect-error -- copyPublicWithBase is honored by the local engine on main but
   // missing from the pinned @takazudo/zfb 0.1.0-next.35 ZfbConfig type (#946); this
   // directive errors (forcing its removal) once the bump to a typed version lands.
+  // Remove together with the TRANSITION copy-public-plugin.mjs fallback (#932).
   copyPublicWithBase: false,
   collections,
   stripMdExt: true,

